src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -18,6 +18,13 @@
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
   </PropertyGroup>
 
+  <!-- default metadata update support for specific platforms -->
+  <PropertyGroup>
+    <MonoMetadataUpdate Condition="'$(MonoMetadataUpdate)' == '' and ('$(TargetsAndroid)' == 'true' or '$(TargetsBrowser)' == 'true')">true</MonoMetadataUpdate>
```

Comment:
Question: `TargetsAndroid` is all Android scenarios, right?  Emulators, too?
I see for iOS there's `TargetsiOS)'` and `TargetsiOSSimulator`